Header header
int16 can_id
string name
int16 define
string msg_data
